src/opm/parser/eclipse/share/keywords/000_Eclipse100/P/PLYROCKM

```diff
@@ -0,0 +1,9 @@
+{"name" : "PLYROCK" , "sections" : ["SCHEDULE"], "size" : {"keyword":"TABDIMS" , "item":"NTSFUN"}, "items":
+    [   {"name":"IPV", "value_type" : "DOUBLE", "dimension":"1" },
```

Comment:
@joakim-hove  Why is PLYROCK added to the SCHEDULE section and what happens when both PLYROCKM and PLYROCK has name PLYROCK?

Reply:
The `PLYROCKM`keyword should actually be in the SCHEDULE section, however the bug here is that there is a misatch between the json file `PLYROCKM` and the internal keyword name `PLYROCK` - a PR updating the latter to `PLYROCK` can be found here: https://github.com/OPM/opm-common/pull/1268